.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: ReleaseCmdStanPy
+
+on:
+  workflow_dispatch:
+    inputs:
+      new_version:
+        description: 'New version, for example: 0.9.69'     
+        required: true
+      old_version:
+        description: 'Old version, for example: 0.9.68'
+        required: true
```

Comment:
Is there a reason we need also the old_version?

In CmdStanPy the version can be found in _version.py file where it is in the line starting with __version__ = 

    __version__ = 'xx.yy.zz'

(there could be also some extra marker (e.g. dev etc)) and comment (#)

Reply:
I've used that to replace old version in `_version.py` but the regex below is much more nicer and doesn't need `old_version` at all so I'll probably just remove it entirely.